Corrected Queue Uri example in code sample

Ankit-MSFT opened the pull request:
corrected Queue URI template as per documentation
https://docs.microsoft.com/en-us/dotnet/api/azure.storage.queues.queueclient.-ctor?view=azure-dotnet#Azure_Storage_Queues_QueueClient__ctor_System_Uri_Azure_Storage_Queues_QueueClientOptions_

# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

Ankit-MSFT committed:
Update Sample01a_HelloWorld.cs

corrected Queue URI template as per documentation

kasobol-msft commented on sdk/storage/Azure.Storage.Queues/samples/Sample01a_HelloWorld.cs:
```diff
@@ -184,7 +184,7 @@ public static void IdentityAuth()
 
             // Create a QueueClient that will authenticate through Active Directory
-            Uri accountUri = new Uri("https://MYSTORAGEACCOUNT.blob.core.windows.net/");
+            Uri accountUri = new Uri("https://MYSTORAGEACCOUNT.queue.core.windows.net/QUEUENAME");
```
could you please change `accountUri` to `queueUri` ?

Ankit-MSFT replied:
hey Kamil, i have done these changes. Thanks.

kasobol-msft reviewed (commented):
Please run `./eng/scripts/Update-Snippets.ps1 storage` . This will update places where this snippet is used and should fix the analyze step that's failing now.

Ankit-MSFT committed:
corrected Uri var name

corrected Uri var name
also ran '.\Update-Snippets.ps1 storage' which updated the README.md file
